apps/web/src/components/Publication/OpenAction/UnknownModule/Tip/index.tsx

```diff
@@ -29,6 +33,13 @@ const TipOpenAction: FC<TipOpenActionProps> = ({
     ? publication?.mirrorOn
     : publication;
 
+  const { data: tipData } = useUnknownOpenActionDataQuery({
+    variables: {
+      module: VerifiedOpenActionModules.Tip,
+      pubId: targetPublication.id
+    }
+  });
```

Comment:
Ok, this is interesting, just found out that it gonna rate limit users. When someone opens a feed and keeps scrolling, it gonna makes 1 extra API call per post. This need to be fixed by somehow using GraphQL alias

Reply:
Going to switch this PR to a draft since the current approach is not feasible due to rate limiting and partial alias support in graphql-codegen. ([more here](https://github.com/heyxyz/hey/pull/4779#discussion_r1541760139))